src/assets/scripts/cri-dockerd.initd

```diff
@@ -0,0 +1,21 @@
+#!/sbin/openrc-run
+
+# This script is used to manage cri-dockerd via OpenRC.
+
+# shellcheck shell=ksh
+
+name="cri-dockerd"
+description="Rancher Desktop Shim for Docker Engine"
+
+
+supervisor=supervise-daemon
+command=/usr/bin/cri-dockerd"
```

Comment:
That `"` looks suspicious — it probably introduces parsing errors? (Tell your editor that this is a shell script to get better syntax highlighting)